### ISSUE TITLE
Number of model parameters

### ISSUE THREAD

**Issue by GoPikachue** [Shaosifan/TransENet#8]
Thank you for your excellent work, I would like to ask if you can provide me with a description of the number of model parameters. When I tested the x4 factor with an image of size (1,3,48,48), I found that it has 37.5 M. I would like to know if this is the correct number of parameters.
   Looking forward to your reply!!!
<img width="230" alt="image" src="https://user-images.githubusercontent.com/57658728/223006828-df8cb074-9a75-49da-a549-cb538b459dab.png">

**Comment by Shaosifan** [comment]
Hi, I just used _thop_ library to compute the number of TransENet and I found that it is almost the same as yours, where the specific number is 37458883. The following code is used to perform the parameter calculations.

```
  from thop import profile
  input = torch.randn(1, 3, 48, 48).to(device)
  macs, params = profile(sr_model, inputs=(input, ))
  print('Params: ' + str(params))
  print('MACs: ' + str(macs))
```

**Comment by GoPikachue** [comment]
Oh! Thanks for the quick answer, I tested the mod using your code and got the same results as you.
   Also, I tested the same parameters as I did with your previous work on HSENet and the results were as follows (Figure 1 for HSENet and Figure 2 for TransENet).
<img width="214" alt="image" src="https://user-images.githubusercontent.com/57658728/223053077-837ade5a-fb77-4567-8967-c00d9d1eb0f3.png">
<img width="176" alt="image" src="https://user-images.githubusercontent.com/57658728/223053193-fa62eb87-d3fe-4e1d-ba32-e8c3a82a9b0f.png">

Thanks this time！
